config/slurm.yml

```diff
@@ -0,0 +1,114 @@
+---
+# This name is used for the Heat stack and as a prefix for the
+# cluster node hostnames.
+slurm_name: slurm
+
+slurm_deploy_user: 'centos'
+
+slurm_net: "caastest-U-internal"
+
+slurm_groups:
+  - "{{ slurm_login }}"
+  - "{{ slurm_compute }}"
+
+slurm_gw_group: "login"
+
+slurm_login:
+  name: "login"
+  flavor: "j1.small"
+  image: "centos-7-20190104"
+  user: "centos"
+  num_nodes: 1
+  root_volume_size: 0
+  node_resource: "Cluster::Node" 
+  extra_groups:
+    - "nfs_clients"
+    - "beegfs_clients"
+  nodenet_resource: "Cluster::NodeNet1WithPreallocatedFIP"
+  nodenet_fips:
+    - uuid: "8723c186-d15f-44d5-9421-e12bf2a03325"
```

Comment:
is there any reason we are specifying both uuid and ip other than ease? wouldn't it be nicer to retrieve the floating ip using the cli from the uuid instead?